# Request empty for now
---
# Name of room to pick-up guest
string pick_up_location

# Name of room to drop-off guest
string drop_off_location

# Guest name, which matches their RFID tag
string guest_name

# Robot start pose
geometry_msgs/Pose robot_start_pose
